demo/common/assets.js

```diff
@@ -764,6 +764,23 @@ shakaAssets.testAssets = [
       .addFeature(shakaAssets.Feature.DASH)
       .addFeature(shakaAssets.Feature.LIVE)
       .addFeature(shakaAssets.Feature.MP4),
+  new ShakaDemoAssetInfo(
+      /* name= */ 'Live sim CEA-608 (VoD)',
```

Comment:
This says both "Live sim" and "Vod", which is it?  Just because it is hosted by the live-sim project doesn't mean you should include that.  A better name would be `DASH-IF CEA-608 VOD` and `DASH-IF CEA-608 Live`.

Reply:
Done!